content/customize-gnome.md

```diff
@@ -18,117 +18,210 @@ section: software
 tableOfContents: true
 ---
 
-The GNOME desktop environment is highly customizable.  With a few extensions, the desktop can look and feel like many other operating system's desktops.  Launcher bars can be added, window lists can be used, and the overall look and feel of the desktop can be changed.
+The GNOME desktop environment is highly customizable. Visit [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed extensions.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
 
-To install a customization, simply head to [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed customizations.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
-
-We will share some of the more popular extensions, but there are many more to choose from!  If one is found and it isn't currently working, or breaks with an update, then try a different extension to see if it works better.
+**NOTE:** There are many extensions to choose from! If one isn't currently working, or breaks with an update, then try a different extension to see if it works better.
 
 ---
 
-## User Themes
+## Included Extensions
 
-[User Themes](https://extensions.gnome.org/extension/19/user-themes/)
+<details>
+  <summary>Click to expand</summary>
 
-This extension allows for the desktop to be themed in further ways than default.  Many themes and extensions can be installed from [opendesktop.org](https://www.opendesktop.org/s/Gnome), among others.  This extension is installed by default in Pop!_OS, and can be added to Ubuntu 18.04.
+Pop!\_OS includes the GNOME ```Extensions``` app by default. Which lists both system-installed, and user-installed extensions.
 
----
+![extensions-app-window](/images/customize-gnome/extensions-app-window.png)
 
-## Dash to Dock
+Each extension has options for customizing settings (if supported in Extension metadata), or viewing the webpage for the extension to learn more information. The Extensions app also allows you to quickly enable and disable installed extensions.
```

Comment:
Where are we capitalizing the singular word "Extension" versus referring to it as an "installed extension?" It seems inconsistent.

Also, you put `Extensions` (the app name) in code backticks earlier, but not here.

Finally, I question the comma usage in the first sentence. "viewing the webpage for the extension" is not another way to say "customizing settings." Using the word "or" there also implies customizing settings and viewing the webpage are mutually exclusive, which isn't the case.

I would suggest:

> Each extension has options for customizing settings (if supported in the extension's metadata) and viewing the webpage for the extension to learn more information. The `Extensions` app also allows you to quickly enable and disable installed extensions.

Reply:
Fixed.